cif/products/src/mappers/PagedResponseProductMapper.ts

```diff
@@ -39,6 +40,11 @@ export default class PagedResponseProductMapper extends Mapper<PagedResponseProd
     } = mappable;
     const facets = selectedFacets.replace(/\|/g, ':');
 
+    const isCIFRequest: boolean = CommerceIntegrationHelper.isCIFRequest(mappable);
```

Comment:
Why do we need a isCIFRequest method, that just validates if filter exists inside the mappable object? you can simply do if(filter)...

Reply:
removed